### DISCUSSION TITLE
3 Streams ? 1 for record, 1 for live view, 1 for motion detect

### DISCUSSION THREAD

**Discussion by bugsbunny88** [blakeblackshear/frigate#9011] [Ask A Question]
I'm a bit confused, and this might totally be wrong . But on the 5442 ; I see Substream 0 ; 1, and 2.... 
I'm not seeing much mention of utilizing all 3 streams for utilizing the cameras capabilities. Sure it might mean 3 connections per camera ; but whats the downside ? or am I missing something inherently.

**Comment by NickM-27** [comment]
I have a few of these cameras and I just use 2 streams
subtype 0 (main stream) 2688x1520 for recording
subtype 2 (sub stream 2) 1280x720 for detection and live view

you can do what you are suggesting but I suggest 1280x720 is a good resolution for detection and for live view
